.editorconfig

```diff
@@ -0,0 +1,2 @@
+indent_style = space
+indent_size = 2
```

Comment:
can we add an EOF newline in here too? 😁

Reply:
Done!